File: a1-softuni-programming-basics-javascript/03b-conditional-statements-advanced-exercise/05-journey.js
function destinationCost(input) {
  let budget = Number(input[0]);
  let season = input[1];
  let vacationPlace = ``;
  let sleepPlace = ``;
  let moneySpent = 0;

  if (budget <= 100) {
    vacationPlace = `Bulgaria`;
    if (season === `summer`) {
      sleepPlace = `Camp`;
      moneySpent = budget * 0.3;
    } else {
      sleepPlace = `Hotel`;
      moneySpent = budget * 0.7;
    }
  } else if (budget <= 1000) {
    vacationPlace = `Balkans`;
    if (season === `summer`) {
      sleepPlace = `Camp`;
      moneySpent = budget * 0.4;
    } else {
      sleepPlace = `Hotel`;
      moneySpent = budget * 0.8;
    }
  } else if (budget > 1000) {
    vacationPlace = `Europe`;
    sleepPlace = `Hotel`;
    moneySpent = budget * 0.9;
  }
  console.log(`Somewhere in ${vacationPlace}`);
  console.log(`${sleepPlace} - ${moneySpent.toFixed(2)}`);
}

File: a01-softuni-programming-basics-python/03b-conditional-statements-advanced-exercise/05_journey.py
total_budget = float(input())
season = input()
destination = ""
accommodation = ""
money_spend = 0


if total_budget <= 100:
    destination = "Bulgaria"
    if season == "summer":
        accommodation = "Camp"
        money_spend = total_budget * 0.3

    elif season == "winter":
        accommodation = "Hotel"
        money_spend = total_budget * 0.7


elif total_budget <= 1000:
    destination = "Balkans"
    if season == "summer":
        accommodation = "Camp"
        money_spend = total_budget * 0.4

    elif season == "winter":
        accommodation = "Hotel"
        money_spend = total_budget * 0.8


else:
    destination = "Europe"
    accommodation = "Hotel"
    money_spend = total_budget * 0.9

print(f"Somewhere in {destination}")
print(f"{accommodation} - {money_spend:.2f}")
